Extension/package.json

```diff
@@ -39,7 +39,31 @@
     "Snippets"
   ],
   "activationEvents": [
-    "*"
```

Comment:
revert this

Reply:
Looks like an extra space is still there. Might be worth running a format document on the package.json file.